### ISSUE TITLE
Illegible text/icons with high contrast themes in Windows

### ISSUE THREAD

**Issue by tomasz1986** [giuspen/cherrytree#359]
With a high contrast theme enabled, all text and background colours are supposed to follow the theme's settings. However, it seems that at least some parts of the CherryTree application for Windows have colours hard coded. As a result, some parts of the UI are simply illegible.

Please check the following screen shots from Windows 10.

High Contrast White - black text on dark purple background on hover
![2](https://user-images.githubusercontent.com/5626656/39057205-a1ce8ccc-44f3-11e8-98f0-aaff2472900f.png)

High Contrast Black - black icons on black background
![1](https://user-images.githubusercontent.com/5626656/39057102-6a9031fc-44f3-11e8-9fd9-86acef94de3d.png)

My question is simple - would it be possible to properly support high contrast themes?

If yes, then please take into account the fact that these themes' colours are 100% user configurable (which is the whole point of accessibility), so hard coding separate colours for High Contrast Black and High Contrast White just to make them "work" is also a big no-no. I am only writing this just in case, because I have seen such a "solution" in other programs.

Below are two M$ sources describing what high contrast themes are and how to support them.
https://msdn.microsoft.com/library/windows/desktop/hh404233%28v=vs.85%29.aspx
https://docs.microsoft.com/windows/uwp/design/accessibility/high-contrast-themes

**Comment by BsNoSi** [comment]
I have accidentally found CherryTree and it seems to be quite useful for me. Playing around with the settings I run into the challenge to change the theme. I name it "challenge" because all attempts to change the theme in Windows does not change anything. 

I would like to switch the fore-/background color but regardless to my choice I have always a black background white font and blue highlighed current paragraph. Where can I change/correct the themes?

**Comment by giuspen** [comment]
It seems you don't have write access to the folder of the preferences, where config.cfg is saved. Check this with the menu item 'Open The Directory with Preferences Files' under 'File' or 'Help' cannot check nor remember right mow

**Comment by BsNoSi** [comment]
Nope. No access rights restrictions on any file in the complete folder.
